soy/pages.soy

```diff
@@ -76,6 +76,7 @@
  */
 {template .passwordSignUp}
   {@param? email: string} /** The email address to prefill. */
+  {@param? requireDisplayName: bool} /** Whether to show the display name. */
```

Comment:
We also have soy tests, in [here](https://github.com/firebase/firebaseui-web/blob/master/soy/pages_test.js#L99) that allows us to display how the template renders given the configuration. You should update the main test to use requireDisplayName: true, and add one with false.